package-lock.json

```diff
@@ -0,0 +1,2529 @@
+{
```

Comment:
This is ignored in the master .gitignore file, may need to do a fresh pull and re-commit the other changes.